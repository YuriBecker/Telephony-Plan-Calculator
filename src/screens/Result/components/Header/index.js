import React from 'react';
import { View } from 'react-native';
import PropTypes from 'prop-types';
import TextTitle from '../TextTitle';
import colors from '../../../../styles/colors';

const index = ({ children }) => {
  return (
    <View
      style={{
        justifyContent: 'center',
        alignItems: 'center',
        backgroundColor: colors.black[1],
        height: '15%',
      }}
    >
      <TextTitle category="h1" style={{ marginBottom: 20, fontWeight: 'bold' }}>
        {children}
      </TextTitle>
    </View>
  );
};

export default index;

index.propTypes = {
  children: PropTypes.node.isRequired,
};
