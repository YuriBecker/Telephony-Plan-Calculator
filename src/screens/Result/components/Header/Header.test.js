import React from 'react';
import { shallow } from 'enzyme';

import Header from './index';

describe('Testing Header component', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<Header>Some text</Header>);
  });

  it('should render correctly', () => expect(wrapper).toMatchSnapshot());
});
