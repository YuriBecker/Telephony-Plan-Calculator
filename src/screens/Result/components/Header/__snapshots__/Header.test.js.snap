// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Header component should render correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#212121",
      "height": "15%",
      "justifyContent": "center",
    }
  }
>
  <index
    category="h1"
    style={
      Object {
        "fontWeight": "bold",
        "marginBottom": 20,
      }
    }
  >
    Some text
  </index>
</View>
`;
